### PR TITLE
docs: added documentation for installing vue-router-next

### PR DESCRIPTION
**Pull request by BboyAkers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
 # vue-router-next [![CircleCI](https://circleci.com/gh/vuejs/vue-router-next.svg?style=svg)](https://circleci.com/gh/vuejs/vue-router-next)
 
+## Install as a package for your vue3 project
+
+```bash
+npm i @posva/vue-router-next --save
```
Comment (posva) [inline README.md:6]: Where did you find these? 😄 it's just `vue-router@next`

**Comment by posva** [comment]
If this is about adding contribution guidelines, we should add them to the contributing.md file. IIRC there are already a few instructions.

About installing vue-router, the instructions should be added to docs (WIP at https://vue-router-next-preview.netlify.app) but it's still a private repo

File: README.md (hunk 1 above)
Comment (BboyAkers) [reply README.md:6]: Ahhh gotcha. I found it here on npmjs. 😆 
https://www.npmjs.com/package/@posva/vue-router-next

**Comment by BboyAkers** [comment]
Gotcha, I'll dig into the contributing.md file to see if I can add anything anywhere and as for the vue-router-next docs. Any idea when that'll be public?

**Pull request closed without merging**